modules/ldap_groups/app/services/ldap_groups/synchronize_groups_service.rb

```diff
@@ -54,7 +54,26 @@ def create_missing!(entries)
       entries.each do |login, data|
         next if existing[login]
 
-        User.try_to_create(data)
+        if OpenProject::Enterprise.user_limit_reached?
```

Comment:
We should place this check inside the `Users::CreateContract` so we don't have to repeat it wherever a user might get created. But this does not have to happen within this PR.

Reply:
Good point, will do that after merged into dev to avoid putting it into 12.0.5

Reply:
I had to do it in this PR after all, as that's what the oteher use of the `try_to_create` method expected in the account registry.